test/test_suite.py

```diff
@@ -42,6 +42,8 @@
 from sqlalchemy.testing.suite.test_select import ExistsTest as _ExistsTest
 from sqlalchemy.testing.suite.test_types import BooleanTest as _BooleanTest
 
+config.test_schema = ""
+
```

Comment:
That's a property to be set by dialect testing system. As Spanner doesn't support named schemas, using an empty str - the default scheme to run tests.